src/lamplib/src/genny/tasks/dry_run.py

```diff
@@ -32,6 +32,9 @@ def dry_run_workload(
         "AuthNInsert.yml",
         "ParallelInsert.yml",
         "LoggingActorExample.yml",
+        "Mixed10KThreads.yml",
```

Comment:
This doesn't seem like a scaleable solution, would it make more sense to get this info directly from the workload file?